spec/mailers/comment_mailer_spec.rb

```diff
@@ -2,6 +2,8 @@
   include MailerSpecHelper
   include EnvVarSpecHelper
 
+  let(:user) { create(:user) }
```

Comment:
we only need this `user` in one spec, why add it here rather than in the spec itself?